.github/workflows/ci.yml

```diff
@@ -17,9 +17,9 @@ jobs:
 
```

Comment:
Also I think the `readme.md` can be updated as well because I think in Prisma v4, mongo is supported officially.

```
PostgreSQL
MySQL
SQLite
MongoDB (Preview) <----- should this be changed ? 
```